specs/signing-and-verification-workflow.md

```diff
@@ -50,7 +50,7 @@ The user wants to pull an OCI artifact only if they are signed by a trusted publ
     1. **Filter signature artifact manifest:**
         1. Filter out the unsupported signature formats by comparing the signature envelope format type (`[descriptors].descriptor.mediaType`) in the signature manifest, with the supported formats defined in [signature specification](./signature-specification.md#storage).
         1. Depending upon the trust-store and trust-policy configuration, further filter out signature manifests.
-            1. Using the `scopes` configured in trust policies, get the applicable trust policy.
+            1. Using the `registryScopes` configured in trust policies, get the applicable trust policy.
```

Comment:
Which PR introduce the change of `scopes` to `registryScopes`?

Reply:
If it is a fix, please also create a PR to the `main` branch.

Reply:
Yeah, we don't have a term called `scopes` in our trust policy. We should fix it in the main branch as well.

Reply:
Created a quick fix PR to main branch: https://github.com/notaryproject/specifications/pull/312.